keyboards/cannonkeys/sagittarius/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = STM32F072
+
+# Build Options
+#   comment out to disable the options.
+#
+
+DFU_SUFFIX_ARGS =
```

Comment:
This is so that this firmware can be flashed on APM32F072CBT6 as well as STM32F072CBT6

Reply:
I believe we came to the conclusion that FFFF for VID and PID works with dfu-util